Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls/Interop/WinRT/WebViewControlProcess.cs

```diff
@@ -60,12 +61,50 @@ private WebViewControlProcess(Windows.Web.UI.Interop.WebViewControlProcess proce
         /// <value><c>true</c> if this instance can access the private network; otherwise, <c>false</c>.</value>
         public bool IsPrivateNetworkClientServerCapabilityEnabled => _process.IsPrivateNetworkClientServerCapabilityEnabled;
 
+        /// <summary>
+        /// Gets a value indicating the partition of the web view.
```

Comment:
Should we mention the version here that this is valid on?  (same below?)

Reply:
That's tricky to know what version it is valid with--the API is versioned at a different cadence than the OS, it just so happens we "snap" and that's what's out there. We handle support by asking the universal contract if a type, property, method, event (etc.) is available, so we don't care what version. Right now the API is visible in insiders fast, insiders slow(ish), and other RS5 builds.